src/applications/vaos/appointment-list/components/RequestedAppointmentDetailsPage.jsx

```diff
@@ -147,7 +147,7 @@ export default function RequestedAppointmentDetailsPage() {
   if (cancelInfo.cancelAppointmentStatus === FETCH_STATUS.failed) {
     return (
       <PageLayout showNeedHelp>
-        <BackLink appointment={appointment} />
+        <BackLink appointment={appointment} showCancelModal />
```

Comment:
I don't think there's a `showCancelModal` argument to `BackLink`

Reply:
I deleted it already.